storybook/src/components/header/header.core-template.ts

```diff
@@ -27,6 +29,8 @@ export const coreHeader: ComponentImplementation<Header> = {
         auth-status=${ifDefined(authStatus)}
         login-url=${ifDefined(loginUrl)}
         logout-url=${ifDefined(logoutUrl)}
+        show-help=${ifDefined(showHelp)}
```

Comment:
Ik denk dat er nu zomaar wel eens `show-help="true"` of `show-help="false"` kan komen te staan.

Stencil maakt van `@Prop() showHelp: boolean` een boolean attribute. En daar heeft Lit weer een speciale syntax voor:

```html
<dso-header ?show-help=${showHelp}>
```

De boolean attribute wordt bij alle truthy values gezet. Hier is geen `ifDefined()` nodig.

Reply:
Dat klinkt cool, dat ga ik zo eens testen :-)